var/spack/repos/builtin/packages/py-darshan/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyDarshan(PythonPackage):
+    """Python utilities to interact with Darshan log records of HPC applications."""
+
+    homepage = "https://www.mcs.anl.gov/research/projects/darshan"
+    pypi = "darshan/darshan-3.4.0.1.tar.gz"
+
+    maintainers = ["jeanbez", "shanedsnyder"]
+
+    version("3.4.0.1", sha256="0142fc7c0b12a9e5c22358aa26cca7083d28af42aeea7dfcc5698c56b6aee6b7")
+
+    depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-importlib-resources", when="^python@3.6", type=("build", "run"))
```

Comment:
I think our test suite has a `pyproject.toml` requirement like this:

`"importlib_resources;python_version<'3.9'"`

Which is a bit different than the canonical usage of `importlib_resources`, so I think it may be required up to that version to get full functionality when testing at least.